snapcraft.yaml

```diff
@@ -1,24 +1,35 @@
 name: classic
-summary: A classic shell environment for Ubuntu Core all-snap installations
-description: This snap provides a full classic shell environment including dpkg and apt support.
-version: 16.04
+summary: A classic shell environment for Ubuntu Core installations
+description: |
+  This snap provides a full classic shell environment including dpkg and
+  apt support.
+version: 18.04-0.1
 confinement: devmode
 architectures: [ all ]
 grade: devel
-assumes: [snapd2.23]
+base: core18
 
 apps:
   create:
     command: bin/create
     plugs: [classic-support]
+    adapter: none
   reset:
     command: bin/reset
     plugs: [classic-support]
+    adapter: none
   classic:
     command: bin/classic
     plugs: [classic-support]
+    adapter: none
 
 parts:
   copy:
     plugin: dump
     source: .
+  base:
+    plugin: nil
+    override-build: |
+      wget -P $SNAPCRAFT_PART_INSTALL http://cdimage.ubuntu.com/ubuntu-base/bionic/daily/current/bionic-base-$(dpkg --print-architecture).tar.gz
```

Comment:
I guess that means all devices, even Ubuntu Core 16, will now be on Bionic. Hmm..

Also, what happens when the snap refreshes?

Reply:
I targetd this to the "18" branch. I should have made this clear in my PR summary, sorry for that. The idea is that you can do `sudo snap install classic --channel=18` and you get bionic. The current "snap install classic" will not change, people still get xenial this way. Not ideal we really need to create a "16" track and move it there so that latest is really latest.